scripts/UPDATE.sh

```diff
@@ -87,8 +95,19 @@ SuperBuild(){
     cd SIRF-SuperBuild
   else
     cd SIRF-SuperBuild
+    git checkout master
```

Comment:
best to replace these 2 with `git fetch`. It won't update the files, just make sure that the local origin is synced with the remote.  The lines below will do then do the appropriate checkout